taskana-adapter-taskana-connector/src/main/java/pro/taskana/adapter/taskanaconnector/api/impl/TaskInformationMapper.java

```diff
@@ -28,6 +29,7 @@
 public class TaskInformationMapper {
 
   private static final Logger LOGGER = LoggerFactory.getLogger(TaskInformationMapper.class);
+  private static final String CAMUNDA_PROCESS_VARIABLE_PREFIX="camunda:";
```

Comment:
I would prefer to have this configurable. But we can do this with another ticket to save time now.

Reply:
btw: is it a good idea to use ':' in a JSON structure?